modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/session/SessionManager.java

```diff
@@ -20,32 +20,73 @@
 import java.util.Map;
 import java.util.UUID;
 import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicBoolean;
+import org.apache.ignite.internal.logger.IgniteLogger;
+import org.apache.ignite.internal.logger.Loggers;
 import org.apache.ignite.internal.sql.engine.CurrentTimeProvider;
+import org.apache.ignite.internal.sql.engine.exec.LifecycleAware;
 import org.apache.ignite.internal.sql.engine.property.PropertiesHolder;
+import org.apache.ignite.internal.thread.IgniteThread;
+import org.apache.ignite.internal.util.worker.IgniteWorker;
 import org.jetbrains.annotations.Nullable;
 
 /**
  * A manager of a server side sql sessions.
  */
-public class SessionManager {
+public class SessionManager implements LifecycleAware {
+    /** Check period in ms. */
+    private static long checkPeriod = TimeUnit.SECONDS.toMillis(10);
```

Comment:
While 10 seconds looks good enough default value, this setting should find its way into the configuration. But for now I would suggest to reduce this to 1 second or even to a half of a second to shorten the time of related tests. WDYT?

Reply:
Also let's move this constant to SqlQueryProcessor and pass it as a constructor argument. This will 1) gather all configuration params in a single place and 2) make it easy to test SessionManager

Reply:
agree, done